src/lm.jl

```diff
@@ -245,6 +245,11 @@ function predict(mm::LinearModel, newx::AbstractMatrix;
     end
     if interval === nothing
         return retmean
+    elseif mm.pp.chol isa CholeskyPivoted &&
+        mm.pp.chol.rank < size(mm.pp.chol, 2)
+        throw(ArgumentError("prediction intervals are currently not implemented " *
+                            "when some independent variables have been dropped " *
+                            "from model due to collinearity"))
```

Comment:
```suggestion
                            "from the model due to collinearity"))
```

Reply:
Glad I'm not the only one to make suggestions about determinants! :-D

Reply:
Perhaps it's a habit I picked up from you at some point 😉

Reply:
Argh..